### PR TITLE
Improve description of channel conflict resolution

### PR DESCRIPTION
**Pull request by c0ffeeca7**
- to match content provided in SkyConnect documentation

### CODE REVIEW

File: src/guides/about-multiprotocol.html [hunk 1]
```diff
@@ -55,12 +55,30 @@ <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
           <li>Re-pair your devices.</li>
         </ol>
       </li>
-      <li><b>Option 2</b>: Use two separate chips, one for each network.
+      <li><b>Option 2</b>: Use two separate wireless chips, one for each network. This works if you have a both Home Assistant
+        Yellow and SkyConnect.
```
Comment (agners) [inline src/guides/about-multiprotocol.html:59]:
```suggestion
      <li><b>Option 2</b>: Use two separate wireless chips, one for each network. E.g. by adding a Home Assistant SkyConnect as a second wireless chip.
```

File: src/guides/about-multiprotocol.html [hunk 2]
```diff
@@ -55,12 +55,30 @@ <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
...
         <ol>
-          <li>On your Home Assistant Yellow, <a href="/guides/disable-multiprotocol/" target="_blank">disable multiprotocol support</a> and restore the Zigbee network.</li>
-          <li>Plug in a SkyConnect and setup a new Thread network there.</li>
+            <li>On your Home Assistant Yellow, <a href="/guides/disable-multiprotocol/" target="_blank">disable
+                    multiprotocol support</a> and restore the Zigbee network.</li>
+            <li>Plug in a SkyConnect and setup a new Thread network there.</li>
+            <li>Go to <a class="my"
+                    href="https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_openthread_border_router"
+                    target="_blank"><b>Add-ons</b></a> and install the experimental <b>Open Thread Border Router</b>
```
Comment (agners) [inline src/guides/about-multiprotocol.html:66]:
```suggestion
                    target="_blank"><b>Add-ons</b></a> and install the experimental <b>OpenThread Border Router</b>
```

File: src/guides/about-multiprotocol.html [hunk 3]
```diff
@@ -55,12 +55,30 @@ <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
...
       </li>
-      <li><b>Option 2</b>: Use two separate chips, one for each network.
+      <li><b>Option 2</b>: Use two separate wireless chips, one for each network. This works if you have a both Home Assistant
+        Yellow and SkyConnect.
         <ol>
-          <li>On your Home Assistant Yellow, <a href="/guides/disable-multiprotocol/" target="_blank">disable multiprotocol support</a> and restore the Zigbee network.</li>
-          <li>Plug in a SkyConnect and setup a new Thread network there.</li>
+            <li>On your Home Assistant Yellow, <a href="/guides/disable-multiprotocol/" target="_blank">disable
+                    multiprotocol support</a> and restore the Zigbee network.</li>
+            <li>Plug in a SkyConnect and setup a new Thread network there.</li>
```
Comment (agners) [inline src/guides/about-multiprotocol.html:63]:
```suggestion
            <li>Plug in a SkyConnect.</li>
```

File: src/guides/about-multiprotocol.html [hunk 4]
```diff
@@ -55,12 +55,30 @@ <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
           <li>Re-pair your devices.</li>
```
Comment (agners) [inline src/guides/about-multiprotocol.html:55]: This is not really related to this change, but still worth changing I think:

```suggestion
          <li>Re-pair your Thread devices.</li>
```

File: src/guides/about-multiprotocol.html [hunk 5]
```diff
@@ -55,12 +55,30 @@ <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
...
+                        <ul>
+                            <li>If you get a message about a missing device, select <b>Edit config</b> and pick the
+                                serial port with SkyConnect.</li>
+                            <li>Select <b>Save</b>.</li>
+                            <li>Start the add-on again.</li>
+                        </ul>
+                    </li>
+                    <li>Your Home Assistant SkyConnect is now set up to work exclusively with Thread. You can now
+                        start adding your Thread devices.</li>
+                </ul>
```
Comment (agners) [inline src/guides/about-multiprotocol.html:79]: Selecting the device will always be necessary when freshly installing the add-on. I would make those steps:

```suggestion
                <li>Select <b>Configuration</b> tab and pick the serial port with SkyConnect.</li>
                <li>Select <b>Save</b>.</li>
                <li>Start the add-on. Your Home Assistant SkyConnect is now set up to work exclusively with Thread.
                     You can now start adding your Thread devices.</li>
```

**Review by agners — CHANGES_REQUESTED** [review]

File: src/guides/about-multiprotocol.html (hunk 1 above)
Comment (c0ffeeca7) [reply src/guides/about-multiprotocol.html]: What are the alternatives to SkyConnect?

File: src/guides/about-multiprotocol.html [hunk 6]
```diff
@@ -52,15 +52,24 @@ <h2 id="resolve-channel-conflict">Resolve channel conflict</h2>
...
-          <li>On your Home Assistant Yellow, <a href="/guides/disable-multiprotocol/" target="_blank">disable multiprotocol support</a> and restore the Zigbee network.</li>
-          <li>Plug in a SkyConnect and setup a new Thread network there.</li>
+            <li>On your Home Assistant Yellow, <a href="/guides/disable-multiprotocol/" target="_blank">disable
+                    multiprotocol support</a> and restore the Zigbee network.</li>
+            <li>Plug in a SkyConnect.</li>
+            <li>Go to <a class="my"
+                    href="https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_openthread_border_router"
+                    target="_blank"><b>Add-ons</b></a> and install the experimental <b>OpenThread Border Router</b>
+                add-on.
+                <li>Select <b>Configuration</b> tab and pick the serial port with SkyConnect.</li>
```
Comment (c0ffeeca7) [inline src/guides/about-multiprotocol.html:67]:
```suggestion
                <li>Select the <b>Configuration</b> tab and pick the serial port with SkyConnect.</li>
```

**Review by agners — APPROVED** [review]
